apify-docs-theme/src/markdown.js

```diff
@@ -95,7 +75,7 @@ const prettifyPRLinks = () => (tree) => {
         const prLinkRegex = /https:\/\/github.com\/[^\s]+\/pull\/(\d+)/g;
         const match = prLinkRegex.exec(node.value);
 
-        if (!match) return;
+        if (!match) return 0;
```

Comment:
I have a hunch this might be why the `apify-sdk-python` docs build is stalling, as [returning an index from the visitor](https://github.com/syntax-tree/unist-util-visit-parents?tab=readme-ov-file#index) means "continue by visiting my n-th sibling", which, with 0, might potentially cause an infinite loop.

Was this a quick eslint's `consistent-return` fix, or did you have a deeper reason for this? 😅

Reply:
> Was this a quick eslint's consistent-return fix, or did you have a deeper reason for this? 😅

Yeah, exactly, feel free to revert that.

Mixed return types are evil, especially in vanilla JS. This is just another proof :D